### PR TITLE
Add travis scripts and specs

### PR DESCRIPTION
**Pull request by giacomolicari**
This PR sets up everything needed by Travis CI to work with our IT infrastructure.

Details in the files within the commit.

### CODE REVIEW

File: travis/deploy_pull_request.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+function publish_pull_request_urls_in_github {
+  REVIEW_FEATURE_URL="https://$PULL_REQUEST_NAME--$REPO_NAME_ALPHANUMERIC.$REVIEW_ENVIRONMENT_DOMAIN"
+
+  # Using the Issues api instead of the PR api
+  # Done so because every PR is an issue, and the issues api allows to post general comments,
+  # while the PR api requires that comments are made to specific files and specific commits
+  GITHUB_PR_COMMENTS=https://api.github.com/repos/${TRAVIS_REPO_SLUG}/issues/${TRAVIS_PULL_REQUEST}/comments
+  curl -H "Authorization: token ${GITHUB_API_TOKEN}" --request POST ${GITHUB_PR_COMMENTS} --data '{"body":"Travis automatic deployment:\r\n '${REVIEW_FEATURE_URL}' \r\n"}'
```
Comment (Velenir) [inline travis/deploy_pull_request.sh:28]: I don't think there's `GITHUB_API_TOKEN` [set in Travis](https://travis-ci.com/github/gnosis/dex-swap/settings)

**Review by Velenir — COMMENTED** [review]
I've added some suggestions in PR #28

File: travis/deploy_pull_request.sh (hunk 1 above)
Comment (giacomolicari) [reply travis/deploy_pull_request.sh]: Now there are GITHUB_API_TOKEN and GITHUB_RELEASE_TAG_API_TOKEN on Travis

**Comment by gnosis-info** [comment]
Travis automatic deployment:
   * **🎩 [Dapp](https://pr27--dexswap.review.gnosisdev.com)**: Testing web app

**Review by Velenir — APPROVED** [review]
PR works in dark mode only, lol
#29

**Comment by Velenir** [comment]
Good work, @giacomolicari !

**Review by anxolin — APPROVED** [review]
🚀

**Pull request merged**